neurom/fst/tests/test_sectionfunc.py

```diff
@@ -100,14 +101,8 @@ def test_section_tortuosity_looping_section():
         (0, 0, 0), (1, 0, 0), (1, 2, 0), (0, 2, 0), (0, 0, 0)
     ])
 
-    # is infinity
-    nt.eq_(_sf.section_tortuosity(sec), np.inf)
-    nt.ok_(math.isinf(_sf.section_tortuosity(sec)))
-    nt.ok_(np.isinf(_sf.section_tortuosity(sec)))
-
-    # is not NaN
-    nt.ok_(not math.isnan(_sf.section_tortuosity(sec)))
```

Comment:
Is it ok to drop `NaN` tests?

Reply:
This function was testing `_sf.section_tortuosity(sec)` against various types of NaN and inf, but since the returned value is np.inf, I feel it was more testing Numpy than NeuroM